testsuite/features/step_definitions/command_steps.rb

```diff
@@ -137,6 +137,9 @@
 end
 
 # Packages
+When(/^I install inter server sync v2 package on server$/) do
+  step %(I install packages "inter-server-sync" on this "server")
+end
```

Comment:
I would avoid wrapping this step, jsut use `I install packages "inter-server-sync" on this "server"` in the scenario please